libs/dyn/value.go

```diff
@@ -46,6 +51,10 @@ func (v Value) Kind() Kind {
 	return v.k
 }
 
+func (v Value) Value() any {
+	return v.v
+}
```

Comment:
Not used.

Reply:
@pietern it is actually used in skipAndOrder to correctly create a new value with location